hivemq-grafana-dashboard-combined.json

```diff
@@ -8310,9 +8310,916 @@
+              "hide": false,
+              "instant": false,
+              "legendFormat": "{{job}} {{route}}",
+              "range": true,
+              "refId": "A"
+            },
+            {
+              "datasource": {
+                "type": "influxdb",
+                "uid": "${InfluxDB}"
+              },
+              "hide": false,
+              "query": "\nfrom(bucket: v.defaultBucket)\n  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)\n  |> filter(fn: (r) => r[\"_measurement\"] =~ /com\\.hivemq\\.extensions\\.mongodb\\.mqtt-to-mongodb(.+)in-progress\\.count\\.current/ and r[\"_measurement\"] != \"com.hivemq.extensions.mongodb.mqtt-to-mongodb.total.in-progress.count.current\" and r[\"_measurement\"] !~ /com\\.hivemq\\.extensions\\.mongodb\\.mqtt-to-mongodb(.+)batches\\.in-progress\\.count\\.current/)\n  |> aggregateWindow(every: ${intervalSeconds}s, fn: mean)\n  |> drop(columns: [\"_start\", \"_stop\", \"_measurement\", \"_field\"])",
+              "refId": "B"
+            }
+          ],
+          "title": "MQTT to MongoDB in progress",
+          "type": "timeseries"
+        },
+        {
+          "datasource": {
+            "type": "datasource",
+            "uid": "-- Mixed --"
+          },
+          "description": "A histogram of the latency of all MQTT to MongoDB messages.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisBorderShow": false,
+                "axisCenteredZero": false,
+                "axisColorMode": "text",
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "axisSoftMin": 0,
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 10,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "insertNulls": false,
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "never",
+                "spanNulls": false,
+                "stacking": {
+                  "group": "A",
+                  "mode": "none"
+                },
+                "thresholdsStyle": {
+                  "mode": "off"
+                }
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": []
+              },
+              "unit": "ms"
+            },
+            "overrides": []
+          },
+          "gridPos": {
```

Comment:
since we don't have batches in progress metrics for mongodb, this is not left aligned, but at the same time it is in the same position as the other latency visualisations, so I'm fine either way